src/main/java/svc/data/citations/filters/FilterCitations.java

```diff
@@ -0,0 +1,35 @@
+package svc.data.citations.filters;
+
+import java.time.LocalDateTime;
+import java.util.List;
+import java.util.ListIterator;
+
+import javax.inject.Inject;
+
+import org.springframework.stereotype.Component;
+
+import svc.managers.CourtManager;
+import svc.models.Citation;
+import svc.models.Court;
+
+@Component
+public class FilterCitations {
+	@Inject
+	CourtManager courtManager;
+
+	public List<Citation> FilterDates(List<Citation> citations){
```

Comment:
This is potentially really inefficient, running a SQL query over to load the court and over when there are multiple responses. It would be better to locally cache each court after loading it, and only reload one from the manager if it's not in the cache.